README.md

```diff
@@ -1,8 +1,12 @@
 # Lunch-Hackers
```

Comment:
Maybe we should consider changing the name. Could be a subject of the first meeting.